src/components/portrait-view.tsx

```diff
@@ -66,6 +67,68 @@ export const PortraitViewRow = (props: PortraitViewRowProps) => {
 
 export const PortraitView = (props: ITableProps) => {
   const {collectionClasses, selectedDataSet, collections, getValueLength} = props;
+  const thresh = useMemo(() => {
```

Comment:
You should just be able to pull this out of the function as a constant since it doesn't depend on any outside values.

Reply:
I was getting an error re recalculating because it is being used in a useEffect.

Reply:
I was getting an error re recalculating because it is being used in a useEffect.